Client/src/utils/TokenService.js

```
@@ -0,0 +1,232 @@
+          /**
+           * Check if the current token is expired
+           * @returns {boolean} True if token is expired or missing, false otherwise
+           */
+          isTokenExpired() {
+            const token = this.getToken();
+            if (!token) return true;
+            
+            const expiryTime = localStorage.getItem(this.tokenExpiryKey);
+            if (!expiryTime) return true;
+            
+            return Date.now() > parseInt(expiryTime);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)